styles/style.css

```diff
@@ -361,3 +358,491 @@ input {
   text-align: center;
   margin-bottom: 5px;
 }
+
+/* Media Queries section for responsive layout mobile version */
+
+@media only screen and (max-width: 600px) {
+  .nav-menu {
+    display: none;
+  }
+
+  .hamburger {
+    display: flex;
+  }
+
+  .grid-container {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 600px) {
+  .nav-menu {
+    display: none;
+  }
+
+  .hamburger {
+    display: flex;
+  }
+
+  .grid-container {
+    display: none;
+  }
+}
+
+/* Media Queries section for responsive layout */
+
+/* ---HEADER-SECTION  desktop--- */
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] **OPTIONAL**
Kindly notice that this is a medium-sized screen, not a desktop-sized one.
You should have followed the project requirements and set up the media query between: (min-width > 992px - 1100px)
This will make the website look good on all screens, not only for the 32 inches monitor. Please look at the image below for more clarification.
You'll find a lot of white space that needs to be fixed.
![Screenshot](https://user-images.githubusercontent.com/47625407/121761295-32232e80-cb2f-11eb-991a-265a3dc6d981.png)

> I'm sorry this didn't show up on the earlier review.

Reply:
okay, I will do all this tomorrow. I didn't see this requirement in the project. simple fix. thank you for sporting it out.